charts/internal/shoot-system-components/charts/csi-driver-manila/templates/daemonset-node.yaml

```diff
@@ -42,22 +42,24 @@ spec:
               add: ["SYS_ADMIN"]
             allowPrivilegeEscalation: true
           image: {{ index .Values.images "csi-driver-manila" }}
-          args:
-            - /bin/manila-csi-plugin
-            - --nodeid=$(NODE_ID)
+          command:
+            - "/bin/sh"
```

Comment:
What is the reason for switching to shell?

Reply:
Because without this the `--nodeaz` command was not being shell-substituted. Instead the whole command string was passed as the `nodeaz` value itself

Reply:
The upstream templates are also using the `/bin/sh` too.

Reply:
ah. right. the curl is executed here.